charts/astronomer/values.yaml

```diff
@@ -18,12 +18,12 @@ images:
     tag: 0.16.3
     pullPolicy: IfNotPresent
   registry:
-    repository: quay.io/astronomer/ap-registry
-    tag: 3.11.5-4
+    repository: astronomerinc/ap-registry
```

Comment:
is this correct?

Reply:
Good catch. This is the old pre-quay registry. This should be reverted to quay.io.